tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -170,6 +170,63 @@ def test_cpu_template(test_microvm_with_ssh, network_config, cpu_template):
                                     "XSAVEC": "false", "XGETBV": "false",
                                     "XSAVES": "false"}
 
+    feature_info_1_edx = {"x87 FPU on chip": "true", "CMPXCHG8B inst": "true",
```

Comment:
For better readability I would split this test in 2 tests:
`test_masked_features` and `test_exposed_features`.

Reply:
done.